nodedb.go

```diff
@@ -166,9 +166,9 @@ func (ndb *nodeDB) Has(hash []byte) (bool, error) {
 	}
 
 	if ldb, ok := ndb.snapshotDB.(*dbm.GoLevelDB); ok {
-		exists, err := ldb.DB().Has(key, nil)
-		if err != nil {
-			return false, err
+		exists, err2 := ldb.DB().Has(key, nil)
```

Comment:
Why was renaming to `err2` necessary here?

Reply:
`shadow: declaration of "err" shadows declaration at line 162 (govet)` I can `nolint` it

Reply:
ah my local setup is more strict then the bot it doesnt seem to be throwing the error